packages/odyssey-react/src/components/TextInput/TextInput.theme.ts

```diff
@@ -13,34 +13,34 @@
 import type { ThemeReducer } from "@okta/odyssey-react-theme";
 
 export const theme: ThemeReducer = (theme) => ({
-  // REMOVE THIS
-  /* eslint-disable @okta/odyssey/no-invalid-theme-properties */
+  BackgroundColor: theme.ColorBackgroundBase,
+  BorderColor: theme.ColorBorderUi,
   BorderRadius: theme.BorderRadiusBase,
   BorderStyle: theme.BorderStyleBase,
   BorderWidth: theme.BorderWidthBase,
-  ColorBackground: theme.ColorBackgroundBase,
-  ColorBackgroundDisabled: theme.ColorBackgroundDisabled,
-  ColorBorder: theme.ColorBorderUi,
-  ColorBorderDisabled: theme.ColorBorderDisabled,
-  ColorBorderHoverFocus: theme.ColorPrimaryBase,
-  ColorBorderInvalid: theme.ColorBorderDangerBase,
-  ColorFocusOutline: theme.ColorFocusPrimary,
-  ColorFocusOutlineInvalid: theme.ColorFocusDanger,
-  ColorText: theme.ColorTextBody,
-  ColorTextDisabled: theme.ColorTextBody,
-  ColorTextPlaceholder: theme.ColorTextSub,
-  FocusOutlineWidth: theme.FocusOutlineWidthBase,
+  DisabledBackgroundColor: theme.ColorBackgroundDisabled,
+  DisabledBorderColor: theme.ColorBorderDisabled,
+  DisabledTextColor: theme.ColorTextBody,
+  /* eslint-disable @okta/odyssey/no-invalid-theme-properties */
+  FocusBoxShadowColor: theme.ColorFocusPrimary,
+  FocusBoxShadowSpread: theme.FocusOutlineWidthBase,
+  /* eslint-enable @okta/odyssey/no-invalid-theme-properties */
```

Comment:
The fact that `FocusBoxShadowColor` ends with `Color` actually allows it to pass our lint rule so we can adjust like so to avoid unnecessarily unlinted code!

```suggestion
  FocusBoxShadowColor: theme.ColorFocusPrimary,
  // eslint-disable-next-line @okta/odyssey/no-invalid-theme-properties
  FocusBoxShadowSpread: theme.FocusOutlineWidthBase,

```

Reply:
addressed in 891a212